### PR TITLE
extension to enable/disable remote signals

### PR DESCRIPTION
**Pull request by AleksanderWWW**
Unit tests should start passing after the revert of this feature is on `neptune-client` `master` branch

EDIT: Done

### CODE REVIEW

File: src/neptune_experimental/run.py [hunk 1]
```diff
@@ -18,16 +18,51 @@
 from typing import Any
```
Comment (Raalsky) [inline src/neptune_experimental/run.py:18]: We have CHANGELOG here as well :D. You can just copy-paste what we had earlier in regular package

Comment (AleksanderWWW) [reply src/neptune_experimental/run.py:18]: My bad, on it! :D

File: src/neptune_experimental/run.py [hunk 2]
```diff
@@ -18,16 +18,51 @@
...
+        if enable_remote_signals is None:
+            self._enable_remote_signals = True  # user did not pass this param in kwargs -> default value
+        else:
+
+            verify_type("enable_remote_signals", enable_remote_signals, bool)
+            self._enable_remote_signals = enable_remote_signals
 
         super().__init__(*args, **kwargs)
+
+    def _prepare_background_jobs(self) -> BackgroundJobList:
```
Comment (Raalsky) [inline src/neptune_experimental/run.py:46]: We're probably able to get rid of this duplicated logic with calling the parent `_prepare_background_jobs` and filtering out the `WebsocketsSignalsBackgroundJob`? What do you think?

Comment (AleksanderWWW) [reply src/neptune_experimental/run.py:39]: Done

File: src/neptune_experimental/run.py (hunk 1 above)
Comment (AleksanderWWW) [reply src/neptune_experimental/run.py:18]: Done

**Review by Raalsky — APPROVED** [review]

**Pull request merged**